.github/workflows/format.yml

```diff
@@ -0,0 +1,24 @@
+name: Format
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+defaults:
+  run:
+    working-directory: javascript
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14.x"
+      - name: Format
```

Comment:
Can this name also be changed to Check Format (Is this just the display name of the check in the GitHub/PR CI? )

Reply:
Yes it is the display name of check. I changed the name of its job to `check_format` for consistency.